backend/.claude/commands/add-env-var.md

```
@@ -0,0 +1,38 @@
+추가할 환경변수명을 인자로 받아 세 파일에 자동으로 삽입한다: `$ARGUMENTS`
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)